### DISCUSSION TITLE
NFT项目防科学家机器人的链上验证码项目

### DISCUSSION THREAD

**Discussion by wartstone** [rebase-network/hello-world#60] [Ideas]
目前很多NFT项目方启动时有很多机器人科学家水帖等，导致大部分白名单被少部分人截取，在考虑是否可以通过链上验证码类似的方式解决此类问题？ 定位是工具类小产品。 有一起想做的吗，欢迎留言讨论

**Comment by luckyyang** [comment]
感觉有些项目有这个需求。链上验证码如何实现呢

**Reply by wartstone** [reply]
目前也在构思，加个微信LS-03COM聊哦
